manifest.json

```diff
@@ -1,40 +0,0 @@
-{
-
-  "manifest_version": 2,
-  "name": "fathom-products",
-  "version": "1.0",
-
-  "description": "Extracts a product from a product detail page.",
-
-  "icons": {
-    "48": "icons/product-icon-48.png",
-    "96":"icons/product-icon-96.png"
-  },
-
-  "page_action": {
```

Comment:
Are the page actions and browser actions used for different things, or what?

Reply:
whoops, left both in by mistake